css/style.css

```
@@ -0,0 +1,215 @@
+/* 
+Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+Font sizes (px)
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+*/
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+.main-wrapper {
+  width: 160rem;
+  margin: 0 auto;
+}
+
+/******************************/
+/* HEADER */
+/******************************/
+
+.promo-header {
+  position: absolute;
+  display: flex;  
+  height: 12rem;  
+  align-items: center;
+  padding-left: 18rem;    
+  gap: 0.8rem;
+}
+
+.header-shadow {
+  width: 160rem;
+  position: absolute;  
+  height: 12rem;    
+  background: linear-gradient(to bottom, black -100%, transparent 90%);  
+}
+
+.logo-menu-bar {
+  display: flex;
+  justify-content: space-between;
+  gap: 22.8rem;
+  align-items: center;
+}
+
+.logo {
+  font-family: "Yeseva One", sans-serif;
+  text-decoration: none;
+  color: #fff;
+  text-transform: uppercase;
+  font-size: 3.6rem;
+  opacity: 90%;
+}
+
+/******************************/
+/* NAVIGATION */
+/******************************/
+
+.main-nav-list {
+  list-style: none;
+  display: flex;
+  gap: 2.6rem;
+}
+
+.main-nav-link,
+.main-nav-link:link,
+.main-nav-link:visited,
+.main-nav-link:active {
+  padding: 4.8rem 1.2rem;
+  position: relative;
+  display: inline-block;
+  text-decoration: none;
+  color: #fff;
+  font-style: normal;
+  font-size: 2.2rem;
+  font-weight: 600;
+  transition: all 0.3s;
+  text-shadow: 1px 1px rgba(0, 0, 0, 0.3);
+  border-top: 0.38rem solid transparent;
+}
+
+.main-nav-link:hover,
+.main-nav-link:active {
+  opacity: 75%;
+  transition: opacity 0.3s;
+}
+
+.main-nav-link.is-active {
+  opacity: 100%;
+  border-top: 0.38rem solid #fff;
+}
+
+.main-nav-link:focus {
+  opacity: 75%;
+  border-top: 0.38rem solid #fff;
+}
+
+/******************************/
+/* SOCIAL-LINKS */
+/******************************/
+
+.social-links {
+  margin-top: 0.8rem;
+  list-style: none;
+  display: flex;
+  gap: 0.8rem;  
+  align-items: center;  
+}
+
+.social-link:link,
+.social-link:visited {
+  display: flex;
+  text-decoration: none;
+  color: #dab33e;
+  transition: all 0.3s;
+  font-size: 2rem;
+  align-items: center;
+  margin: 0.2rem;  
+}
+
+.social-link:hover {
+  opacity: 75%;
+  transition: opacity 0.3s; 
+}
+
+/******************************/
+/* PROMO SECTION */
+/******************************/
+
+.promo-banner {
```

It looks like whole section "promo" has this background
```suggestion
.promo-section {
```

---

Fixed